README.md

```diff
@@ -27,6 +27,38 @@ This repository has the unified data required to perform user and username enume
 
 See [CONTRIBUTING](CONTRIBUTING.md)
 
+# Command Line Arguments
+There are quite a few command line options available
+
+`Check for the users yooper and maxim, defaults to outputing json to stdout, only returns the found results.`
```

Comment:
Can you change all these entries so only the python command and switches are in `fixed-width/code` font? For example:

Check for the users yooper and maxim, defaults to outputing json to stdout, returns the sites where no matches were found.
```python whats_my_name.py -u yooper maxim -n```

Reply:
I changed the formatting as suggested.